### PR TITLE
helper step to check for number of replicas for a cluster resource

### PR DESCRIPTION
**Pull request by pruan-rht**
example usage `Given there are 6 "daemon_set" replicas in the "openshift-monitoring" project`.  Should help with PR #2029

@lihongyan1 this should help with your PR

### CODE REVIEW

File: features/step_definitions/replicas.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  clazz = resource_class(resource_type)
+  if BushSlicer::ProjectResource > clazz
+    list = clazz.list(user: user, project: project(namespace))
+  else
+    list = clazz.list(user: user)
+  end
+  list.each do | cl |
+    desired_count = cl.replica_counters(cached: false)[:desired]
+    if desired_count != expected.to_i
+      raise "#{resource_type} count is #{desired_count}, but is expected to be #{expected} instead"
```
Comment (lihongyan1) [inline features/step_definitions/replicas.rb:17]: Better change as the following to provide more information to merge
 raise "#{resource_type} #{cl.name} count is #{desired_count}, but is expected to be #{expected} instead"

Comment (pruan-rht) [reply features/step_definitions/replicas.rb]: @lihongyan1 fixed and edited for grammar error as well.
```      daemon_set node-exporter count is 1, but is expected to be 2 instead (RuntimeError)```

**Comment by pruan-rht** [comment]
@liangxia PTAL and merge if there is no objections as @lihongyan1's PR https://github.com/openshift/verification-tests/pull/2029  is dependent on this

**Pull request merged**